Use aerosol_is instead of is_bulk

gold2718 opened the pull request:
I need to run some tests but this is my entry-level version of what can be done now and I think should be okay as is.

Commit:
Use aerosol_is instead of is_bulk

fvitt commented:
@gold2718 This looks okay to me.  I will do a quick test on ACOM's modeling2 using your fork/branch.

fvitt commented:
I am having some compile errors.  Would I be able to push to your fork/branch?  Otherwise, I could merge in your mods as is and fix the issues in my branch..

gold2718 commented:
> I am having some compile errors. Would I be able to push to your fork/branch? Otherwise, I could merge in your mods as is and fix the issues in my branch..

I sent you an invite to my fork (in case you need it to push).

fvitt committed:
fix the model_is method

	modified:   src/chemistry/aerosol/aerosol_properties_mod.F90
	modified:   src/chemistry/aerosol/bulk_aerosol_properties_mod.F90
	modified:   src/chemistry/aerosol/carma_aerosol_properties_mod.F90
	modified:   src/chemistry/aerosol/modal_aerosol_properties_mod.F90
